docs/recipe/shopware.md

```diff
@@ -34,26 +36,36 @@
 ### repository
 [Source](https://github.com/deployphp/deployer/search?q=%22repository%22+in%3Afile+language%3Aphp+path%3Arecipe+filename%3Ashopware.php)
 
+* Overrides [`repository`](/docs/recipe/common.md#repository) from `recipe/common.php`
```

Comment:
@peterjaap maybe remove set('repository')?

Reply:
Yeah possible, but I'd rather leave it in to remind people of what to do (for those less familiar with Deployer)